lib/fastboot-response.js

```diff
@@ -0,0 +1,20 @@
+var cookie = require('cookie');
+var FastBootHeaders = require('./fastboot-headers');
+
+function FastbootResponse(response) {
+  this.headers = new FastBootHeaders(response.headers);
+  this.statusCode = 200;
+}
+
+FastbootResponse.prototype.cookie = function (cookieString) {
```

Comment:
I'm down with that. I know we talked about it but if forgot what the consensus was when I was writing it. Will remove.